src/modules/messages/components/assistant-message.tsx

```
@@ -71,16 +71,18 @@ export default function AssistantMessage({
-                className="prose prose-invert text-sm max-w-none w-full prose-img:my-4 prose-pre:my-4 prose-table:my-4 prose-hr:my-4 prose-th:border prose-td:border prose-table:border-collapse prose-th:p-2! prose-td:p-2!"
-                suppressHydrationWarning
-                dangerouslySetInnerHTML={{ __html: cleanHtmlContent }}
-              ></div>
-            </div>
+            <>
+              <div ref={messageStartRef}></div>
+              <div className="bg-zinc-900 border border-zinc-800 text-white px-4 py-3 rounded-2xl rounded-bl-md shadow-lg">
+                <div
+                  className="prose prose-invert text-sm max-w-none w-full prose-img:my-4 prose-pre:my-4 prose-table:my-4 prose-hr:my-4 prose-th:border prose-td:border prose-table:border-collapse prose-th:p-2! prose-td:p-2!"
+                  suppressHydrationWarning
+                  dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)